include/openthread/border_routing.h

```diff
@@ -106,6 +106,17 @@ typedef struct otBorderRoutingPrefixTableEntry
     uint32_t          mPreferredLifetime;   ///< Preferred lifetime of the on-link prefix when `mIsOnLink` is true.
 } otBorderRoutingPrefixTableEntry;
 
+/**
+ * This structure represents the prefix infomation received from the platform.
+ *
+ */
+typedef struct otBorderRoutingPlatformOmrPrefixInfo
+{
+    otIp6Prefix mPrefix;
+    uint32_t    mPreferredRemainingMs;
+    uint32_t    mValidRemainingMs;
+} otBorderRoutingPlatformOmrPrefixInfo;
```

Comment:
Can consider re-using `otBorderRoutingPrefixTableEntry` for this (has the same fields)?